src/view.rs

```diff
@@ -54,6 +54,51 @@ impl View for Label {
     }
 }
 
+#[derive(Debug)]
+pub struct Spinner(pub(crate) KeyOrValue<Color>);
+
+impl Spinner {
+    /// Create a spinner widget
+    pub fn new() -> Spinner {
+        Spinner(theme::LABEL_COLOR.into())
+    }
+
+    /// Builder-style method for setting the spinner's color.
+    ///
+    /// The argument can be either a `Color` or a [`Key<Color>`](druid::Key).
+    pub fn with_color(mut self, color: impl Into<KeyOrValue<Color>>) -> Self {
+        self.0 = color.into();
+        self
+    }
+
+    /// Set the spinner's color.
+    ///
+    /// The argument can be either a `Color` or a [`Key<Color>`](druid::Key).
+    pub fn set_color(&mut self, color: impl Into<KeyOrValue<Color>>) {
+        self.0 = color.into();
+    }
```

Comment:
I would personally prefer just having a `color` method. Then it can be used as
```rust
Spinner::new().color(my_fancy_color).build(cx);
```
and we only need one method for it.

Reply:
That makes sense. I can change that easily.